### DISCUSSION TITLE
system autopatch and reboot (if reboot is required)

### DISCUSSION THREAD

**Discussion by fritz0011** [uyuni-project/uyuni#8249] [General]
Hello, 

Is anyone that can point to the right direction...:
- we need to implement a kind of automated patch and restart (if any of the applied updates require reboot)
python script or through chained action in web interface....

**Comment by mcalmer** [comment]
Check "recurring actions" feature with custom states. Especially the state which is shipped with uyuni called "uptodate"

**Reply by fritz0011** [reply]
Sooper, 
+ have any fix for ubuntu  (even if system shows restart required, uyuni web still not shows the icon/info that the system require reboot)

**Comment by fritz0011** [comment]
++ great thanks to explain how da heck that custom quartz time is working :) 
some better examples ....
